reconcile/utils/oc.py

```diff
@@ -1705,12 +1773,9 @@ def clusters(
             return list(cluster_map.keys())
         return [k for k, v in cluster_map.items() if v]
 
-    def cleanup(self):
-        for oc in self.oc_map.values():
-            if oc:
-                oc.cleanup()
-        for oc in self.privileged_oc_map.values():
-            if oc:
+    def cleanup(self) -> None:
+        for oc in list(self.oc_map.values()) + list(self.privileged_oc_map.values()):
```

Comment:
can avoid create new list by using `chain`

```python
for oc in itertools.chain(sself.oc_map.values(), self.privileged_oc_map.values()):
```

or another loop:

```python
for m in [self.oc_map, self.privileged_oc_map]:
  for oc in m.values():
```

Reply:
👍 changed in [01d56cf](https://github.com/app-sre/qontract-reconcile/pull/5167/commits/01d56cff88729682f7b0695a88e63fa9448fb483)